provider/win_utils.py

```diff
@@ -0,0 +1,73 @@
+import os
+
```

Comment:
And how about moving all virtio function to win_driver_installer_test.py, is there any concern?

Reply:
> And how about moving all virtio function to win_driver_installer_test.py, is there any concern?

Hello @xiagao , after I've checked functions left in the win_utils.py, I thought those two functions left may be NOT related to the win_driver_installer_test.py ( since the file name is win driver as prefix ).
What's your opinion?
Thanks.

Reply:
Hi @nickzhq , about the left two functions, I thought there is no need to keep in a single win_utils.py because:
1.The two functions are kind of other functions you move to win_driver_installer_test.py, I suggest to make them together.
2.  Except installer test scenario,no other test scenario would call it right now.

Thanks.